CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+### template
```

Comment:
does this mean, for each version show the following list? i think it would be better to just start the changelog with the first release